MAUI/Linear-Gauge/scale.md

```diff
@@ -302,7 +302,7 @@ SfLinearGauge gauge = new SfLinearGauge();
 
 ## Customize scale values
 
-Linear gauge allows you to display a set of values along with a custom scale based on your business logic by using the `GenerateVisibleLabels`,`ValueToFactor`, and `FactorToValue` callbacks.
+Linear gauge allows you to display a set of values along with a custom scale based on your business logic by using the [`GenerateVisibleLabels`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SfLinearGauge.html#Syncfusion_Maui_Gauges_SfLinearGauge_GenerateVisibleLabels),[`ValueToFactor`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SfLinearGauge.html#Syncfusion_Maui_Gauges_SfLinearGauge_ValueToFactor_System_Double_), and [`FactorToValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SfLinearGauge.html#Syncfusion_Maui_Gauges_SfLinearGauge_FactorToValue_System_Double_) callbacks.
```

Comment:
provide space between "," and "ValueToFactor"

Reply:
provided sir